### ISSUE TITLE
configure: error: C++ compiler cannot create executables

### ISSUE THREAD

**Issue by mikroskeem** [mikroskeem/musl_root#10]
```
configure:2749: checking for C++ compiler default output file name
configure:2771: g++    conftest.cpp  >&5
/tools/lib/gcc/x86_64-unknown-linux-musl/8.2.0/../../../../x86_64-unknown-linux-musl/bin/ld: cannot find -lstdc++
collect2: error: ld returned 1 exit status
configure:2775: $? = 1
configure:2813: result: 
configure: failed program was:
| /* confdefs.h.  */
| #define PACKAGE_NAME "LZMA"
| #define PACKAGE_TARNAME "lzma"
| #define PACKAGE_VERSION "4.32.7"
| #define PACKAGE_STRING "LZMA 4.32.7"
| #define PACKAGE_BUGREPORT "lasse.collin@tukaani.org"
| #define NDEBUG 1
| #define PACKAGE "lzma"
| #define VERSION "4.32.7"
| /* end confdefs.h.  */
| 
| int
| main ()
| {
| 
|   ;
|   return 0;
| }
configure:2819: error: C++ compiler cannot create executables
See `config.log' for more details.
```

Solution: install libstdc++

**Comment by mikroskeem** [comment]
Note: I should use xz anyway but this doesn't change the fact that C++ is broken

**Comment by mikroskeem** [comment]
Fixed in 0fd0de0e
